label_studio/tasks/models.py

```diff
@@ -255,7 +255,6 @@ class Annotation(models.Model):
     objects = AnnotationManager()
     tracker = FieldTracker(fields=['ground_truth', 'result'])
 
-    state = JSONField('state', null=True, default=dict, help_text='Editor state (system data)')
```

Comment:
why?

Reply:
Because we don't need it for more over a year

Reply:
I meant that we can remove later all useless stuff in one migration, no rush for that now